Pipfile.lock

```diff
@@ -18,10 +18,10 @@
     "default": {
         "certifi": {
             "hashes": [
-                "sha256:339dc09518b07e2fa7eda5450740925974815557727d6bd35d319c1524a04a4c",
-                "sha256:6d58c986d22b038c8c0df30d639f23a3e6d172a05c3583e766f4c0b785c0986a"
+                "sha256:47f9c83ef4c0c621eaef743f133f09fa8a74a9b75f037e8624f83bd1b6626cb7",
```

Comment:
can we run this with `--keep-outdated`? (i.e. so we won't have the dependency updates in Pipfile.lock)

Reply:
@kushaldas wanted us to *not* update the dependencies at this time. I think we should just do it now anyway. Figure we should have some agreement on this before we do it?